pangeo_forge/tasks/zarr.py

```diff
@@ -1,17 +1,23 @@
+from typing import List, Optional
+
 import fsspec
 import zarr
 from prefect import task
 
 
 @task
-def consolidate_metadata(target):
+def consolidate_metadata(target, writes: Optional[List[str]] = None) -> None:
     """
-    Consolidate Zarr metadata
+    Consolidate the metadata the Zarr group at `target`.
 
     Parameters
     ----------
     target : str
-         Path or url of the Zarr store.
+        The URL for the (combined) Zarr group.
+    writes : list of strings, optional
```

Comment:
I'm not sure how prefect runs through it's DAG, but `consolidate_metadata` didn't have a direct dependency on the previous writes, which would maybe(?) let it run before the writes.